src/fu-plugin.c

```diff
@@ -551,13 +552,57 @@ fu_plugin_get_dmi_value (FuPlugin *plugin, const gchar *dmi_id)
 	return fu_hwids_get_value (priv->hwids, dmi_id);
 }
 
+/**
+ * fu_plugin_get_dmi_string:
+ * @plugin: A #FuPlugin
+ * @structure_type: A SMBIOS structure type
+ * @offset: A SMBIOS offset
+ *
+ * Gets a hardware SMBIOS string.
+ *
+ * Since: 0.9.8
+ **/
+const gchar *
+fu_plugin_get_dmi_string (FuPlugin *plugin, guint8 structure_type, guint8 offset)
```

Comment:
I'm not crazy about the mixup of dmi and smbios in the naming here.  Could you just keep everythign consistent to make it fu_plugin_get_smbios_string?

Reply:
Agreed, changed.